index.html

```diff
@@ -42,5 +42,132 @@ <h1 class="down_word">I’m a software developer</h1>
             <br>
         </section>
     </section>
+    <section class="work">
+        <h1 class="upper_word"><br>My Recent <br> Works</h1>
+        <img src="icon/IndicatorIndi.png" alt=""  class="IndicatorIndi">
+        
+        <nav class="grid-container">
+            <nav class="grid-item one"></nav>
+            <nav class="grid-item main">
+                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
+            </nav>
+            
+            <nav class="grid-item flex-container">
+                <nav class="flex-item flex-item1"><h4>Ruby on Rails</h4> </nav>
+                <nav class="flex-item flex-item2"><h4>CSS</h4></nav>
+                <nav class="flex-item flex-item3"><h4>JavaScrit </nav>
+                <nav class="flex-item flex-item4"><h4>HMTL</h4></nav>
+            </nav>
+
+            <nav class="grid-item last">
+                    <button class="button1" type="button">See project</button>
+            </nav>
+        </nav>
+        <nav class="grid-container">
+            <nav class="grid-item one"></nav>
+            <nav class="grid-item main">
+                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
+            </nav>
+            
+            <nav class="grid-item flex-container">
+                <nav class="flex-item flex-item1"><h4>Ruby on Rails</h4> </nav>
+                <nav class="flex-item flex-item2"><h4>CSS</h4></nav>
+                <nav class="flex-item flex-item3"><h4>JavaScrit </nav>
+                <nav class="flex-item flex-item4"><h4>HMTL</h4></nav>
+            </nav>
+
+            <nav class="grid-item last">
+                    <button class="button1" type="button">See project</button>
+            </nav>
+        </nav>
+        <nav class="grid-container">
+            <nav class="grid-item one"></nav>
+            <nav class="grid-item main">
+                    <h3>Multi-Post Stories <br>Gain+Glory</h3>
+            </nav>
+            
+            <nav class="grid-item flex-container">
+                <nav class="flex-item flex-item1"><h4>Ruby on Rails</h4> </nav>
+                <nav class="flex-item flex-item2"><h4>CSS</h4></nav>
+                <nav class="flex-item flex-item3"><h4>JavaScrit </nav>
+                <nav class="flex-item flex-item4"><h4>HMTL</h4></nav>
+            </nav>
+
+            <nav class="grid-item last">
+                    <button class="button1" type="button">See project</button>
+            </nav>
+        </nav>
+        
+    </section>
+    <section class="about">
+            <nav class="grid-container-about">
+                <nav class="First-image">
+                    <img src="icon/illustration 1- about me-mobile.svg" class="upper_img">
```

Comment:
- [ ] This image is a background image. Best practice to add a background image is to use `background-image` property of CSS, not the HTML `img` tag. Kindly fix this issue.

Reply:
Thanks a lot, I did it!